js/models/messages.js

```diff
@@ -2191,8 +2182,12 @@
         // Show that the session reset is "in progress" even though we had a valid session
         this.set({ endSessionType: 'ongoing' });
       }
-
-      if (message.isSessionRequest()) {
+      /**
+       * A session request message is a friend-request message with the flag
+       * SESSION_REQUEST set to true.
+       */
+      const flag = textsecure.protobuf.DataMessage.Flags.SESSION_REQUEST;
```

Comment:
this should be renamed to something more descriptive than `flag`. Maybe just name it `sessionRequestFlag`?

Reply:
I renamed that one and a few other used the same way